xsd/src/schema_manager.js

```diff
@@ -237,7 +237,9 @@ SchemaManager.prototype.exportNamespaces = function() {
 	var self = this;
 	var namespacePrefixes = {};
 	var prefixUsed = {};
+	this.addNamespacePrefixes(namespacePrefixes, prefixUsed, {'xs': self.xsNS});
```

Comment:
Please also have a look at the respective commit messages (emphasis on "in final schema JSON"):

* a861932cb071dd98b61664c6ac70e1780445da25
* 852e112a26b1df11d6c478db58e92c0b9bf7b590